br: deprecate rawkv backup and restore (#9733)
pingcap/docs: br/rawkv-backup-and-restore.md

# Back Up and Restore RawKV

> **Warning:**
>

> > This feature is deprecated since TiDB v6.2.0 and no support will be provided thereafter.

Reviewer: ```suggestion
> This feature is deprecated since TiDB v6.2.0 and no support will be provided since TiDB v6.4.0.
```

> TiKV can be independent of TiDB, and work with PD to provide KV databases. Such a product including only TiKV and PD is called RawKV. Backup & Restore (BR) supports data backup and restore for products that use RawKV. This document describes how to back up and restore RawKV.

Reviewer 2: ```suggestion
TiKV and PD can constitute a KV database when used without TiDB, which is called RawKV. Backup & Restore (BR) supports data backup and restore for products that use RawKV. This document describes how to back up and restore RawKV.
```
